db/keyval/etcd/plugin_impl_etcd.go

```diff
@@ -154,6 +183,43 @@ func (plugin *Plugin) Compact(rev ...int64) (toRev int64, err error) {
 	return 0, fmt.Errorf("connection is not established")
 }
 
+// If ETCD is connected, complete all other procedures
+func (plugin *Plugin) configureConnection(etcdCfg *Config) {
+	plugin.reconnectResync = etcdCfg.ReconnectResync
```

Comment:
I think it would be better to add `*Config` field to the plugin instead of just manually setting the fields.